### DISCUSSION TITLE
SFTP: Delete folder at once

### DISCUSSION THREAD

**Discussion by frspp** [iterate-ch/cyberduck#15210] [Q&A]
**Is your feature request related to a problem? Please describe.**
I connected to SFTP server and deleted folders containing thousands of files. Each file is deleted separately so it takes a lot of time. 

**Describe the solution you'd like**
Delete the folder itself like rm -rf does.

**Describe alternatives you've considered**
SSH'ing to server and deleting the folder with command rm -rf

**Comment by dkocher** [comment]
Unfortunately the SFTP specification does not allow recursive delete of files.
